src/property.ts

```diff
@@ -60,10 +66,12 @@ export const DEFAULT_PROPERTY_PRECENCE: Property[] =  [
 
   // Nested Property
   Property.BIN_MAXBINS,
+  Property.SCALE_TYPE,
```

Comment:
In hindsight, this `SCALE_TYPE` should come after `SCALE`.
Otherwise we might get duplicates of outputs that have `scale=false`